modules/javafx.graphics/src/test/java/test/javafx/stage/Popup_owner_Test.java

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) 2011, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Either we should use `git mv <oldFile> <newFile>` here to retain the file commit history.
or
As the test changed a lot from before, we could choose not to do `git mv`, in the case the copyright in new file should be only 2025

I would recommend to `git mv` as a general good practice.

Reply:
This was done with `git mv`. The problem is that the similarity between the old file and new file was too little for Git to consider this to be an actual rename, even when using `git mv`. I believe the similarity index threshold in Git is 50%.

The only way to achieve this would be to split it into two separate commits - rename, then rewrite - but that will still disappear since Skara will squish all the commits in a PR into one.